.tx/config

```diff
@@ -2,17 +2,9 @@
 host = https://www.transifex.com
 lang_map = fa_AF: prs, uz@Cyrl: uz, uz@Latn: uz_Latn
 
-[app-menu-management.menu-management-pot]
-file_filter = src/i18n/<lang>.po
+[menu-management-app.en-pot]
```

Comment:
I'm not sure we want to rename this, as it might create a duplicate entry in Transifex (@Philip-Larsen-Donnelly)

Reply:
Well, we it's ok to change the file name, we can easily remove the "old" one later when we decide we don't need it (it does not harm, but may cause confusion).

However there is a bigger problem here :) 
The identifier is in the format `[<project slug>.<file-slug>]`. The project slug is "app-menu-management" and you cannot simply change that as "menu-management-app" project doesn't exist in transifex - so the sync will not work with the change as it is currently.